src/main/java/org/opentripplanner/api/model/ApiStop.java

```diff
@@ -3,18 +3,19 @@
 
 import java.io.Serializable;
 import java.util.Objects;
+import org.opentripplanner.util.I18NString;
 
 public class ApiStop implements Serializable {
     private static final long serialVersionUID = 1L;
 
     public String id;
-    public String name;
+    public I18NString name;
```

Comment:
These should be `String`, and we should do locale-aware mapping.

Reply:
Fixed.